### DISCUSSION TITLE
Wallpaper Source Please!

### DISCUSSION THREAD

**Discussion by Sachinanand99** [end-4/dots-hyprland#663] [General]
Can you provide me the wallpaper of the first pic in readme.md. Please

**Comment by ryfenri** [comment]
here i found it
![flower](https://github.com/end-4/dots-hyprland/assets/116497111/d8191e75-fbfd-487b-891e-eb65d319a1ef)

**Reply by Sachinanand99** [reply]
Thanks 😊
